packages/e2e-tests/tests/react_devtools-04-seeking.test.ts

```diff
@@ -0,0 +1,44 @@
+import { openDevToolsTab, startTest } from "../helpers";
+import {
+  getReactComponents,
+  jumpToMessageAndCheckComponents,
+  waitForReactComponentCount,
+} from "../helpers/new-react-devtools-panel";
+import { getTimelineCurrentPercent, seekToTimePercent } from "../helpers/timeline";
+import test, { expect } from "../testFixtureCloneRecording";
+
+test.use({ exampleKey: "cra/dist/index_chromium.html" });
+
+test("react_devtools-04: Component selection is maintained when seeking to a new point", async ({
+  pageWithMeta: { page, recordingId },
+}) => {
+  const queryParams = new URLSearchParams();
+  // Enable the new RDT panel
+  queryParams.set("features", "feature_newReactDevTools");
+
+  await startTest(page, recordingId, undefined, queryParams);
+  await openDevToolsTab(page);
+
+  // Seek to the first console message and check that the initial tree has 3 components
+  await jumpToMessageAndCheckComponents(page, "Initial list", 3);
+
+  const initialTimePercent = await getTimelineCurrentPercent(page);
+
+  // Select the root component
+  const firstComponent = getReactComponents(page).nth(0);
+  await firstComponent.click();
+  await expect(firstComponent).toHaveAttribute("data-selected", "true");
```

Comment:
nit: We could use either a `getSelectedReactComponent` helper or a `verifyReactComponentIsSelected` helper for this.